jwt/api_jwt.py

```diff
@@ -100,6 +98,18 @@ def _validate_claims(self, payload, options, audience=None, issuer=None,
                           'Please use verify_exp in options instead.',
                           DeprecationWarning)
 
+        deprecated_requires = ['require_exp', 'require_iat', 'require_nbf']
+        verify_claims = set(deprecated_requires).intersection(options)
```

Comment:
It seems kind of silly to initialize a set here on every call. Why not initialize this set as a property on the `PyJWT` class so it happens once?

Also, are you sure doing a set intersection here is actually faster then simply doing something like:
```python
verify_claims = [requires for requires in deprecated_requires if requires in options]
```

In some cases (small sets of items), it may be faster to just do something like the above. I'm not saying it is in this case, but you should find out.